### ISSUE TITLE
What is EdgeAttentionLoss? and how do i use it?

### ISSUE THREAD

**Issue by Birdylx** [PaddlePaddle/PaddleSeg#993]
I notice that there is a edge_attention_loss.py in the repo, and as the doc string described, it only compute the crossentropy loss on edge part. It really confused me, what is edge part? what is the shape of the logit and label in multi-class segmentation task? is there any paper describe the edge attention loss?

**Comment by michaelowenliu** [comment]
@Birdylx Hi, that loss is from decoupled segnet, https://github.com/PaddlePaddle/PaddleSeg/blob/release/v2.0/configs/decoupled_segnet/decoupledsegnet_resnet50_os8_cityscapes_832x832_80k.yml
https://github.com/PaddlePaddle/PaddleSeg/blob/release/v2.0/paddleseg/models/decoupled_segnet.py

**Comment by Birdylx** [comment]
@michaelowenliu thanks for your quick reply, i will look into it.
